### DISCUSSION TITLE
Lookup does not get new data

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6316] [Help]
Hi,

I have this lookup and I manually call at Dialog file but every time this execute it returns same data.

![2022-03-11 22_28_02-Window](https://user-images.githubusercontent.com/13443821/157887121-bd55788b-0fb0-4b68-9fc3-5f83e7772579.png)

I added Expiration (1, -1, 2, etc.) at LookupScript but still not working.
![2022-03-11 22_32_33-Window](https://user-images.githubusercontent.com/13443821/157887535-59ba7ece-669c-424b-b9a0-220d53f5240f.png)

Help?

**Comment by minhhungit** [comment]
try with Expiration = -1 (not 1)
and call Q.ReloadLookup before using

**Reply by axelcasauran** [reply]
it works when I use "Q.getLookup<Administration.StartingNumberRow>('Administration.StartingNumberLookup')"
`var parentRow = Q.getLookup<Administration.StartingNumberRow>('Administration.StartingNumberLookup').items.filter(x => x.StrModule == "Accounting" && x.StrTransactionType == "General Journal");`
